packages/webapp/pages/onboarding.tsx

```diff
@@ -231,6 +245,14 @@ export function OnboardPage(): ReactElement {
       return setActiveScreen(OnboardingStep.PWA);
     }
 
+    if (
+      extensionExperiment &&
+      shouldShowExtensionOnboarding &&
+      activeScreen !== OnboardingStep.Extension
```

Comment:
This will work counter intuitive if PWA or AndroidApp are also on right?
It's fine if they can never be on at same time, but we need to be super clear about that.

Reply:
Nvm linter removes because we already have the early return for check :|